src/lib/item-types.js

```diff
@@ -128,6 +124,20 @@ const itemTypes = {
       }
     }
   },
+  'taxonomy': {
```

Comment:
are both of these necessary? are you going to resolve this TODO before submission?

Reply:
These two are used in `ScoreSetEditor` page. `taxonomy` is used in `taxonomies` and `taxonomy-search` is used to get the whole taxonomies and list them in dropdown menu. 
No, I'm not going to solve the TODO before submission. It's from Jeremy's initial codes in this page. I'm not sure what he prefers to modify this part in the future so just mark them as others.